lab_2_retrieval_w_bm25/main.py

```diff
@@ -18,6 +18,17 @@ def tokenize(text: str) -> list[str] | None:
 
     In case of corrupt input arguments, None is returned.
     """
+    if not isinstance(text, str):
+        return None
+    text = text.lower()
+    cleaned_text = ""
+    for symbol in text:
+        if symbol.isalpha() or symbol == " " or symbol == "\n":
```

Comment:
why `\n` ?

Reply:
I don't know, I feel that way, because there is \n in the texts, but it seems nothing changes if I remove this condition